app/mailers/referee_mailer.rb

```diff
@@ -3,6 +3,7 @@ def reference_request_email(reference)
     @application_form = reference.application_form
     @reference = reference
     @candidate_name = @application_form.full_name
+    @candidate_first_name = @application_form.first_name
```

Comment:
Do you want to just make a `candidate_first_name` helper method in the mailer?  I don't feel strongly about it.

Reply:
I do actually prefer the existing pattern here where it's clear exactly what is used in the email at a glance. I appreciate its a little bit of duplication, but we are only using it three times. 

It also broke a test and didn't think it was worth spending more time on it.